server/app/views/templates/library/index.scala.html

```diff
@@ -125,6 +125,7 @@ <h2 class="pull-left">@section.name.humanizeUnderscore</h2>
       <!-- JS -->
     <script src="@routes.Assets.at("lib/highlightjs/highlight.min.js")" type="text/javascript"></script>
     <script src="@routes.Assets.at("lib/highlightjs/languages/scala.min.js")" type="text/javascript"></script>
+    <script src="@routes.Assets.at("javascripts/emojify-cdn.js")"></script>
```

Comment:
What do you think about including this plugin with webjars `"org.webjars.npm" % "emojify.js" % "1.1.0"` ? @dialelo looks good to you?